framework/src/modules/interoperability/types.ts

```diff
@@ -16,6 +16,7 @@ import { Logger } from '../../logger';
 import { APIContext, EventQueue } from '../../node/state_machine';
 import { SubStore } from '../../node/state_machine/types';
 
+export type getStoreFunction = (moduleID: number, storePrefix: number) => SubStore;
```

Comment:
As per style guides type names are advised to be PascalCase, meaning `GetStoreFunction` is better here. Also using a verb in type name is a bit off I think, we can maybe say `StoreCallback`?

Reply:
And I am sure you know these things so I think you should incorporate more rest into your working hours and don't work while you are sleepy dear Ishan 🙂  Sleep is so important 😴